editor/plugins/sprite_frames_editor_plugin.h

```diff
@@ -75,14 +78,26 @@ class SpriteFramesEditor : public HSplitContainer {
 	ConfirmationDialog *delete_dialog;
 
 	ConfirmationDialog *split_sheet_dialog;
-	ScrollContainer *splite_sheet_scroll;
+	ScrollContainer *split_sheet_scroll;
```

Comment:
This typo is still present in the `master` so not sure if I should be fixing it in here.

Reply:
Are you sure it's supposed to be `splite` -> `split`, or is it `splite` -> `sprite`?

Reply:
I'm not sure, I didn't think about this change. For sure all of these are supposed to refer to the same thing:
https://github.com/godotengine/godot/blob/1e762e939f810451e4807cdfed46feff14e1fc0c/editor/plugins/sprite_frames_editor_plugin.h#L80-L88

Reply:
Seems like there's also this unused member:
https://github.com/godotengine/godot/blob/1e762e939f810451e4807cdfed46feff14e1fc0c/editor/plugins/sprite_frames_editor_plugin.h#L62
But same as above, not sure if I should remove it in this PR.

Reply:
From what I can gather from the original PR, "split" seems to be the correct answer. And if that `split` variable is truly not used anywhere, it should be removed.

Reply:
> And if that `split` variable is truly not used anywhere, it should be removed.

Removed. Fun fact: seems like you're the one who forgot to remove it in #25692. :wink:

Reply:
Damn you, `git blame`! But seriously, good catch.